ivy/functional/frontends/numpy/statistics/order_statistics.py

```diff
@@ -104,3 +108,29 @@ def nanpercentile(
                     arrayofpercentiles.append(_cpercentile(ii, i))
                 resultarray.append(arrayofpercentiles)
         return resultarray
+
+
+
+@ivy.numpy_frontend
+@ivy.handle_numpy_out
+@ivy.handle_numpy_dtype
+@ivy.to_ivy_arrays_and_back
+@ivy.handle_numpy_casting
+@ivy.from_zero_dim_arrays_to_scalar

```

Comment:
I'm not quite sure why it is necessary to add the ivy. prefix when you have already imported it into the current namespace